server/controllers/inventory/depots/index.js

```diff
@@ -208,12 +213,20 @@ function searchByName(req, res, next) {
       LEFT JOIN sector s ON s.uuid = v.sector_uuid
       LEFT JOIN province p ON p.uuid = s.province_uuid
       LEFT JOIN country c ON c.uuid = p.country_uuid
+      LEFT JOIN depot_permission dp  ON dp.depot_uuid = d.uuid
```

Comment:
I think you need to remove these.  There isn't a need do a left join here and only causes repeats in this list.

![image](https://user-images.githubusercontent.com/896472/94271058-fca2ca80-ff38-11ea-99e1-8dc02a41d4c3.png)